openwisp_monitoring/device/migrations/__init__.py

```diff
@@ -90,36 +86,3 @@
         "contents": "#!/bin/sh\ntouch /etc/crontabs/root\n/etc/init.d/cron start\n/usr/sbin/legacy-update-openwisp-packages\n/usr/sbin/legacy-openwisp-monitoring\n",  # noqa
     }
 )
-
-
-def assign_permissions_to_groups(apps, schema_editor):
-    create_default_permissions(apps, schema_editor)
-    operators_read_only_admins_manage = [
-        'devicedata',
-        'devicemonitoring',
-        'wificlient',
-        'wifisession',
-    ]
-    manage_operations = ['add', 'change', 'delete']
-    Group = get_swapped_model(apps, 'openwisp_users', 'Group')
-
-    try:
-        admin = Group.objects.get(name='Administrator')
-        operator = Group.objects.get(name='Operator')
-    # consider failures custom cases
-    # that do not have to be dealt with
-    except Group.DoesNotExist:
-        return
-
-    for model_name in operators_read_only_admins_manage:
-        try:
-            permission = Permission.objects.get(codename='view_{}'.format(model_name))
-            operator.permissions.add(permission.pk)
-        except Permission.DoesNotExist:
-            pass
-        for operation in manage_operations:
-            admin.permissions.add(
-                Permission.objects.get(
-                    codename='{}_{}'.format(operation, model_name)
-                ).pk
-            )
```

Comment:
why is this being moved?

Reply:
`assign_permissions_to_groups()` is placed both in `device/migrations/__init__.py` and `monitoring/migrations/__init__.py`
and one of them never get used in our code.